examples/images/diffusion/README.md

```diff
@@ -1,28 +1,25 @@
 # ColoDiffusion: Stable Diffusion with Colossal-AI
 
-*[Colosssal-AI](https://github.com/hpcaitech/ColossalAI) provides a faster and lower cost solution for pretraining and
```

Comment:
I guess this is because my previous submission is outdated and this is the latest version I git pull from the official repo....